tests/foreman/ui/test_host.py

```diff
@@ -17,46 +18,72 @@ def test_create_host(self):
         @Assert: Host is created
 
         """
+        host = entities.Host()
+        host.create_missing()
+        os_name = u'{} {}'.format(
```

Comment:
I've done this on purpose after [this](https://github.com/SatelliteQE/robottelo/pull/2335#discussion_r31431190) comment. So... @Ichimonji10 , @JacobCallahan what style is better? :)